schema/crdb/dbinit.sql

```diff
@@ -1339,9 +1339,15 @@ CREATE TABLE IF NOT EXISTS omicron.public.oximeter (
     time_created TIMESTAMPTZ NOT NULL,
     time_modified TIMESTAMPTZ NOT NULL,
     ip INET NOT NULL,
-    port INT4 CHECK (port BETWEEN 0 AND 65535) NOT NULL
+    port INT4 CHECK (port BETWEEN 0 AND 65535) NOT NULL,
+    time_deleted TIMESTAMPTZ
```

Comment:
I think you should probably ignore this but: I get a little squirmy about using the word "deleted" for things that aren't an abstraction where the user on the other side of it (usually a human) thinks they're removing it.  It's often not clear what "deleted" means -- e.g., someone might confuse this for thinking the Oximeter instance is expunged (which it probably is, but that's not what this field means).  Does "deleted" here mean "no longer eligible for taking on new assignments" or "this thing's producers should be reassigned" or both?  What about "time_retired"?

Reply:
> Does "deleted" here mean "no longer eligible for taking on new assignments" or "this thing's producers should be reassigned" or both?

Both, and also "no longer included in the list of Oximeter instances" (which is kind of a restatement of the first one, since assignments are chosen by listing Oximeter instances).

`time_retired` is appealing, but I'm a little hesitant to introduce a new word. Is `time_expunged` terrible? The thing that sets this to a non-NULL value is blueprint execution when it sees that the zone has been expunged.

Reply:
`time_expunged` seems reasonable to me.

Reply:
:+1: renamed in [1bdb670](https://github.com/oxidecomputer/omicron/pull/6518/commits/1bdb6704146266a3a1a5f5df5f9e056e11d43809)